CI: Fix an internal issue where aiohttp expected errors were not retried.
